llpc/translator/lib/SPIRV/SPIRVUtil.cpp

```diff
@@ -47,9 +47,26 @@ void addFnAttr(LLVMContext *Context, CallInst *Call, Attribute::AttrKind Attr) {
 #if LLVM_MAIN_REVISION && LLVM_MAIN_REVISION < 396596
   // Old version of the code
   Call->addAttribute(AttributeList::FunctionIndex, Attr);
-#else
-  // New version of the code (also handles unknown version, which we treat as latest)
+#elif LLVM_MAIN_REVISION && LLVM_MAIN_REVISION < 440909
```

Comment:
I don't think any changes are needed in this file, because this function is never called with any of the memory-effect attributes.